.github/workflows/e2e.yml

```diff
@@ -12,15 +12,27 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
+        cable_driver: ['libreswan', 'wireguard']
         deploytool: ['operator']
         globalnet: ['', 'globalnet']
-        cable_driver: ['libreswan', 'wireguard']
+        k8s_version: ['1.17.17']
         ovn: ['', 'ovn']
         exclude:
           - ovn: 'ovn'
             globalnet: 'globalnet'
           - ovn: 'ovn'
             cable_driver: 'wireguard'
+        include:
+          # Recentness of K8s versions are limited by kindest/node image releases
+          - k8s_version: 1.18.15
+          - k8s_version: 1.19.7
+          - k8s_version: 1.20.2
+          - k8s_version: 1.20.2
+            cable_driver: 'wireguard'
+          - k8s_version: 1.20.2
+            globalnet: 'globalnet'
+          - k8s_version: 1.20.2
+            ovn: 'ovn'
```

Comment:
```suggestion
```

These combinations should be checked with the default version we're using..

Reply:
I added an inline comment to clarify that these are temporary, to provide coverage for latest-K8s version with the major features implemented here, and will go away once default=latest version (but we shouldn't do that until we have these checks running to let us know what's coming).